### ISSUE TITLE
SessionID set-cookie doesn´t work when using electron

### ISSUE THREAD

**Issue by johanneslundin** [cypress-io/cypress#5508]
### Current behavior: 

sessions cookie is not set so not possible to bypass the login screen of my application.

### Desired behavior:

Just like when using chrome, i want the `sessionID` to be set when using electron so i can run in headless

### Steps to reproduce: (app code and test code)

```js
Cypress.Commands.add(
  'loginRequest',
  (orgId) => {
    cy.request('POST', 'http://auth.xx.test:4000/oauth/login', {
      email: Cypress.env('email'),
      password: Cypress.env('password'),
      redirectUri: 'http://beta.xx.test:4200',
    }).then(response => {
      cy.request('POST', 'http://auth.xx.test:4000/oauth/token', {
        code: response.body.authorizationCode,
        grantType: 'authorizationCode',
        redirectUri: 'http://beta.xx.test:4200',
      });
    });
  }
);

beforeEach(function () {
    cy.loginRequest(3);
    cy.visit('xx/accounts');
});
```

No cookies are sent. when using chrome `sessionsID` was set by the http://auth.xx.test:4000/oauth/token request using `set-cookie` in the response header but this have stopped working. 

![image](https://user-images.githubusercontent.com/43027282/67749329-37e90080-fa2d-11e9-87db-21ffccb474c4.png)

### Versions

Cypress 3.5.0
electron 73

**Comment by johanneslundin** [comment]
> Hey @johanneslundin, you mentioned `but this have stopped working.` Are you saying that this previously worked in version 3.4.1? That would be helpful in narrowing down the issue.

Yes, but i just manage to fix it. Previously i didn´t have to specify the domain when i set the cookie but now when i dit, it worked. It did something like this.

```
Cypress.Commands.add(
  'loginRequest',
  (orgId) => {
    cy.request('POST', 'http://auth.xx.test:4000/oauth/login', {
      email: Cypress.env('email'),
      password: Cypress.env('password'),
      redirectUri: 'http://beta.xx.test:4200',
    }).then(response => {
      cy.request('POST', 'http://auth.xx.test:4000/oauth/token', {
        code: response.body.authorizationCode,
        grantType: 'authorizationCode',
        redirectUri: 'http://beta.xx.test:4200',
      }).then((response)=> {
cy.setCookie(response.headers['set-cookie'][0], {domain:'.xx.test'})
});
    });
  }
);
```

**Comment by flotwig** [comment]
I think this change in behavior will be fixed by #5478.

Though, in `3.5.0`, `3.6.0`, and #5478, doing `cy.setCookie('foo', 'bar')` from the domain `foo.bar.com` always sets the cookie on `.bar.com`, so I was unable to reproduce the issue @johanneslundin mentions.

Also, @johanneslundin, cy.request() should set the cookie on the browser for you, there should be no need to do the cy.setCookie(). There are some bugs with this in the latest release that will be resolved when #5478 .

Closing for now.
